### PR TITLE
Fixing mismatch between audio and labels and exposing fft parameters for hp search

### PR DESCRIPTION
**Pull request by femke-sintef**

### CODE REVIEW

File: training/lightning_trainer/trainingmodule.py [hunk 1]
```diff
@@ -40,8 +40,9 @@ def finetune_function(self, pl_module: pl.LightningModule, epoch: int, optimizer
 
 class TransferTrainingModule(pl.LightningModule):
 
-    def __init__(self, learning_rate, num_target_classes):
+    def __init__(self, learning_rate, num_target_classes, model_arguments=None):
```
Comment (frafra) [inline training/lightning_trainer/trainingmodule.py:43]: `model_arguments={}` would avoid having to write additional if/else statements afterwards.

File: utils/parsing_utils.py [hunk 2]
```diff
@@ -1,40 +1,43 @@
...
-    if len(filename) > 2:
-        filename = ".".join(filename[0:-1])
-    else:
-        filename = input.split("/")[-1].split(".")[0]
+def remove_extension(input):
+    """Removes filename extension AND directory path"""
+    filename = Path(input)
+    filename = filename.with_suffix("")
+    filename = os.path.basename(filename)
+    return str(filename)
```
Comment (frafra) [inline utils/parsing_utils.py:11]: `os.path.basename` returns a string. No need to call `str` here.

Comment (femke-sintef) [reply utils/parsing_utils.py:11]: agreed. It just lingered there because I first didn't have the basename line, as I hadn't realised the function was supposed to do more than just remove the extension

File: training/lightning_trainer/trainingmodule.py (hunk 1 above)
Comment (femke-sintef) [reply training/lightning_trainer/trainingmodule.py:43]: No, don't do this! See the important warning here: https://docs.python.org/3/tutorial/controlflow.html#default-argument-values.  Having a mutuable as a default value will cause chaos

Comment (frafra) [reply training/lightning_trainer/trainingmodule.py:43]: I fail to see how it would affect `__init__`.
It also seems to be dead code.

Comment (femke-sintef) [reply training/lightning_trainer/trainingmodule.py:43]: Good point. I didn't commit my line 46 since I had to change the pretrained path to run it outside docker. However, for this model_arguments part to do anything it should be changed to:

`self.aclp = AudioCLIP(pretrained=f'/app/assets/AudioCLIP-Full-Training.pt', **model_arguments)`
